pkg/controllers/uiplugin/plugin_info_builder.go

```diff
@@ -108,7 +112,13 @@ func PluginInfoBuilder(plugin *uiv1alpha1.UIPlugin, pluginConf UIPluginsConfigur
 			return pluginInfo, nil
 		}
 	case uiv1alpha1.TypeTroubleshootingPanel:
-		return createTroubleshootingPanelPluginInfo(plugin, namespace, plugin.Name, image, []string{})
+		{
+			pluginInfo, err := createTroubleshootingPanelPluginInfo(plugin, namespace, plugin.Name, image, []string{})
+			pluginInfo.Korrel8rImage = pluginConf.Images["korrel8r"]
+			pluginInfo.LokiServiceNames[OpenshiftLoggingNs], _ = getLokiServiceName(ctx, k, OpenshiftLoggingNs)
+			pluginInfo.LokiServiceNames[OpenshiftNetobservNs], _ = getLokiServiceName(ctx, k, OpenshiftNetobservNs)
```

Comment:
Don't skip error handling here.

Reply:
Added the error check

Reply:
@periklis can you mark this resolved? The code is fixed, and this is the last blocker.